inst/stan/functions/spatial_deviation_noise_matrix_rng.stan

```diff
@@ -1,30 +1,17 @@
 /**
   * Generates a matrix where columns are random vectors for spatial process.
-  * Currently this is for an exponential correaltion function.
-  * @param dist_matrix Distance matrix for correlation constr.
-  * @param sigma_eps Parameter for construction of covariance matrix
-  * of spatial epsilon.
-  * @param phi Phi parameter for exponential decay corr. func.
-  * @param l l paraamter for exponential decay corr. func.
+  * @param Sigma_matrix Matrix of covariances, constructed externally.
```

Comment:
Would still name this `sigma_matrix` bc of linting...

Reply:
The linter will expect that everything will be `snake_case`. I think it only checks the R files but when you pull this in as a variable in R will want it to not be flagged